client/my-sites/domains/domain-management/name-servers/wpcom-nameservers-toggle.jsx

```diff
@@ -20,18 +22,18 @@ class NameserversToggle extends PureComponent {
 	render() {
 		return (
 			<Card compact className="name-servers__dns">
-				<span className="name-servers__title">
-					{ this.props.translate( 'Use WordPress.com Name Servers' ) }
-				</span>
-
 				<form className="name-servers__toggle">
 					<ToggleControl
 						id="wp-nameservers"
 						name="wp-nameservers"
 						onChange={ this.handleToggle }
 						checked={ this.props.enabled }
 						value="active"
+						label={ this.props.translate( NAMESERVERS_TOGGLE_TITLE ) }
+						className="name-servers__title"
+						onClick={ this.handleToggle }
 					/>
+					<ScreenReaderText>{ this.props.translate( NAMESERVERS_TOGGLE_TITLE ) }</ScreenReaderText>
```

Comment:
Furthermore, we can see that both times when `NAMESERVERS_TOGGLE_TITLE` is used, it's in the same component, and even in the same component.

So why not just declare a `const toggleTitle = translate( 'Use WordPress.com Name Servers' );` at the top of the `render()` function and use the `toggleTittle` variable in both places?

Reply:
This was a major major ahaaa moment, Thank you. 🔥🔥